packages/paste-website/src/pages/components/date-picker/index.mdx

```diff
@@ -0,0 +1,617 @@
+            <ListItem>Disabled: $shadow-border-weak</ListItem>
+            <ListItem>Error: $shadow-border-error</ListItem>
+            <ListItem>Error hover: $shadow-border-error-strong</ListItem>
+          </UnorderedList>
+        </Td>
+        <Td>No</Td>
+      </Tr>
+      <Tr>
+        <Td>Background</Td>
+        <Td>
+          <UnorderedList marginBottom="space0">
+            <ListItem>Default: $color-background-body</ListItem>
+            <ListItem>Disabled, Readonly: $color-background</ListItem>
+          </UnorderedList>
+        </Td>
+        <Td>No</Td>
+      </Tr>
+      <Tr>
+        <Td>Value text</Td>
+        <Td>
+          <UnorderedList marginBottom="space0">
+            <ListItem>$font-size-30</ListItem>
+            <ListItem>Default, Readonly: $color-text</ListItem>
+            <ListItem>Disabled: $color-text-weaker</ListItem>
+          </UnorderedList>
+        </Td>
+        <Td>No</Td>
+      </Tr>
+      <Tr>
+        <Td>Help text</Td>
+        <Td>$color-text-weak, $font-size-30</Td>
+        <Td>No</Td>
+      </Tr>
+      <Tr>
+        <Td>Inline error</Td>
+        <Td>
+          <UnorderedList marginBottom="space0">
+            <ListItem>Text: $color-text-error, $font-size-30</ListItem>
+            <ListItem>Icon: IconError, $color-text-error, $icon-size-20</ListItem>
+          </UnorderedList>
+        </Td>
+        <Td>No</Td>
+      </Tr>
+      <Tr>
+        <Td>Spacing</Td>
+        <Td>
+          <UnorderedList marginBottom="space0">
+            <ListItem>
+              Label:
+              <UnorderedList marginBottom="space0">
+                <ListItem>Bottom: $space-10</ListItem>
+                <ListItem>Between label and required indicator: $space-20</ListItem>
+              </UnorderedList>
+            </ListItem>
+            <ListItem>
+              DatePicker:
+              <UnorderedList marginBottom="space0">
+                <ListItem>Left padding: $space-40</ListItem>
+                <ListItem>Top, right, bottom padding: $space-30</ListItem>
+              </UnorderedList>
+            </ListItem>
+            <ListItem>
+              Help text:
+              <UnorderedList marginBottom="space0">
+                <ListItem>Top: $space-20</ListItem>
+              </UnorderedList>
+            </ListItem>
+            <ListItem>
+              Inline error:
+              <UnorderedList marginBottom="space0">
+                <ListItem>Spacing between icon and text: $space-20</ListItem>
+                <ListItem>Top: $space-20</ListItem>
+              </UnorderedList>
+            </ListItem>
+          </UnorderedList>
+        </Td>
+        <Td>No</Td>
+      </Tr>
+    </TBody>
+  </Table>
+</Box>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/date-picker - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {DatePicker, Label, HelpText, formatReturnDate} from '@twilio-paste/core';
```

Comment:
Let's change the import line here to reference the unbarreled packages, like:
```
import {DatePIcker} from '@twilio-paste/core/date-picker';
etc
```
More deets: https://deploy-preview-1498--paste-docs.netlify.app/getting-started/engineering/#how-the-core-package-stays-lean